commands/command_filter_process.go

```diff
@@ -125,6 +350,16 @@ func statusFromErr(err error) string {
 	return "success"
 }
 
+// delayedStatusFromErr returns the status code that should be sent over the
+// filter protocol based on a given error, "err" when the blob smudge operation
+// was delayed.
+func delayedStatusFromErr(err error) string {
+	if err != nil && err != io.EOF {
+		return "error"
+	}
+	return "delayed"
```

Comment:
Good call. I was hesitant to write this in the way you suggested originally, because it binds the implementation of this function to the protocol having the value "success". That said, I think this change is well motivated, and I am going to follow up with a future pull request to make these values non protocol-specific.